style/style.css

```diff
@@ -152,3 +154,263 @@ body {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 0;
+  gap: 1.2rem;
+  width: 32.7rem;
+  height: 30.7rem;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.post-heading {
+  font-size: 3.2rem;
+  height: 36px;
+  font-family: 'Crete Round', Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 44px;
+  display: flex;
+  align-items: center;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.post-description {
+  width: 32.7rem;
+  height: 14.7rem;
+  font-family: 'Inter', Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: #091e42;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.tag-list {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 0;
+  gap: 12px;
+  width: 297px;
+  height: 40px;
+  flex: none;
+  order: 2;
+  flex-grow: 0;
+}
+
+.tag-items {
+  list-style-type: none;
+  font-size: 1.5rem;
+  font-weight: 500;
+  padding: 1rem 1.2rem;
+  border: 1px solid var(--color-border);
+}
+
+.seeProject-button {
+  text-decoration: none;
+  color: var(--color-white);
+  padding: 1.2rem;
+  font-size: 1.7rem;
+  font-weight: 700;
+  background-color: var(--color-sccent);
+  flex: none;
+  order: 3;
+  flex-grow: 0;
+}
+
+.work-card {
+  width: 32.7rem;
+  height: 38.6rem;
+  display: flex;
+  margin-bottom: 2.4rem;
+  flex-direction: column;
+  background-image: url(../images/project-background.svg);
+}
+
+.info-holder {
+  display: flex;
+  flex-direction: column;
+  padding: 0 1.6rem;
+  row-gap: 1.2rem;
+  margin-top: auto;
+}
+
+.project-heading {
+  font-family: 'Crete Round', Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.card-description {
+  width: inherit;
+  font-family: 'Inter', Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 1.6rem;
+  line-height: 2.4rem;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.card-feature-items {
+  list-style-type: none;
+  font-size: 1.5rem;
+  font-weight: 500;
+  padding: 1rem 1.2rem;
+  background-color: rgba(255, 255, 255, 0.2);
+}
+
+.button-section {
+  width: 32.7rem;
+  height: 4.8rem;
+  display: flex;
+  background: var(--color-sccent);
+  justify-content: center;
+  align-items: center;
+  margin-top: 2.2rem;
+}
+
+.card-button {
+  text-decoration: none;
+  font-size: 1.7rem;
+  font-weight: 700;
+}
+
+.white-font {
+  color: var(--color-white);
+}
+
+.background-black {
+  background-color: var(--color-primary);
+  row-gap: 2.4rem;
+}
+
+.about-me-section {
+  margin: 12rem 0;
+}
+
+.accent-color {
+  color: var(--color-sccent);
+}
+
+.about-me {
+  font-size: 6.2rem;
+  padding-top: 12rem;
+}
+
+.get-my-resume {
+  padding: 1.2rem;
+  background-color: var(--color-sccent);
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  width: fit-content;
+}
```

Comment:
- [ ] Also add the hover effect to the "Get my resume" button since this is a requirement.
![image](https://user-images.githubusercontent.com/29071984/170829745-a87d3bb1-c4c8-4aad-adbb-c7b2b4fe2775.png)